packages/prisma/src/prisma/migrations/20231010100715_element_additions_and_practice_quiz/migration.sql

```diff
@@ -0,0 +1,149 @@
+-- CreateEnum
+CREATE TYPE "PublicationStatus" AS ENUM ('DRAFT', 'PUBLISHED');
+
+-- CreateEnum
+CREATE TYPE "ElementOrderType" AS ENUM ('SEQUENTIAL', 'SHUFFLED', 'LAST_RESPONSE');
+
+-- CreateEnum
+CREATE TYPE "ElementInstanceType" AS ENUM ('PRACTICE_QUIZ');
+
+-- CreateEnum
+CREATE TYPE "ElementStackType" AS ENUM ('LIVE_QUIZ', 'PRACTICE_QUIZ', 'MICROLEARNING', 'GROUP_ACTIVITY');
+
+-- AlterTable
+ALTER TABLE "Element" ADD COLUMN     "version" INTEGER NOT NULL DEFAULT 1;
+
+-- AlterTable
+ALTER TABLE "QuestionResponse" ADD COLUMN     "elementInstanceId" INTEGER,
+ALTER COLUMN "questionInstanceId" DROP NOT NULL;
```

Comment:
If we just create a new column and drop the old one don't we lose all the data then?